src/training/calculator/CalculatorOperationSelect.java

```diff
@@ -0,0 +1,52 @@
+package training.calculator;
+
+/**
+ * Created by Ivan_Stepantsov on 7/16/2017.
+ */
+class CalculatorOperationSelect {
+    CalculatorOperationSelect(String operationSelect) {
+        int result;
+        int operandOne;
+        int operandTwo;
+
+        ValuesScanner valuesScanner = new ValuesScanner();
+        operandOne = valuesScanner.getFirstValueSelected();
+        operandTwo = valuesScanner.getSecondValueSelected();
+
```

Comment:
actually, as i've remember - i've did that at the first time, but something went wrong.. now works fine looks like.